src/main/java/com/rcv/ResultsWriter.java

```diff
@@ -235,15 +245,10 @@ private void generateSummarySpreadsheet(
       String candidateDisplayName = this.config.getNameForCandidateID(candidate);
       csvPrinter.print(candidateDisplayName);
 
-      // displayRound indexes over all rounds plus final results round
-      for (int displayRound = 1; displayRound <= numRounds + 1; displayRound++) {
-        // flag for the last round results which are special-cased
-        boolean isFinalResults = displayRound == numRounds + 1;
-        // For the Final Results "round", we're mostly copying the data from the final round.
-        // round from which to display data
-        int dataUseRound = isFinalResults ? numRounds : displayRound;
+      // displayRound indexes over all rounds
+      for (int displayRound = 1; displayRound <= numRounds; displayRound++) {
```

Comment:
now we can just rename this back to `round`

Reply:
yep